extensions/google-java/src/main/kotlin/com/connectrpc/extensions/JavaErrorParser.kt

```diff
@@ -31,30 +33,22 @@ internal object JavaErrorParser : ErrorDetailParser {
      * expects a generic type E? and the underlying type is a Google
      * Java Message.
      */
-    @Suppress("UNCHECKED_CAST")
     override fun <E : Any> unpack(any: AnyError, clazz: KClass<E>): E? {
-        val value = any.value.utf8().decodeBase64() ?: any.value
```

Comment:
This was really confusing how some things were stored as `ByteString`, but actually were intended to be base64-encoded strings. This was also a source of problems -- somewhere along the way, things were not correctly decoded, or perhaps were inadvertently getting double-decoded, because error details didn't actually work.

So I've changed it so that the raw JSON model has a base64-encoded string. And every other representation is the decoded bytes. So there's no base64-decoding in this method anymore.

Also, this was building an `Any`, and then deserializing, but it seemed simpler to just directly deserialize from the bytes.